src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,97 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Locale;
+
 public class SalaryInfo {
+    private static final int quantityOfSubstrings = 4;
+    private Date dateFrom;
+    private Date dateTo;
+    private SimpleDateFormat format;
+    private ArrayList<EmployeeData> employeeData;
```

Comment:
same here - let's have only constants as class fields
this is a service class that should not store any state